docs/source/_templates/sdk_overview_tutorial.rst

```diff
@@ -88,7 +88,8 @@ to create transactions and submit them as :term:`Sawtooth batches<Batch>`.
 {% else %}
 
    For a full Python implementation of the XO transaction family, see
-   ``/{project}/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
+   `https://github.com/hyperledger/sawtooth-sdk-python/tree/add-sdk/examples/xo_python
+   <https://github.com/hyperledger/sawtooth-sdk-python/tree/add-sdk/examples/xo_python>`_.
```

Comment:
This link doesn't work.  This is the link I see in the sawtooth-sdk-python repo: https://github.com/hyperledger/sawtooth-sdk-python/tree/master/examples/xo_python

A bigger issue is whether it's correct to "hard code" the XO example links to master. That's not for this PR to handle - it does the right thing by matching the other SDK example links - so I'll make a general comment elsewhere.